src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -218,7 +218,17 @@ const	modules = {
 			url: "/spa?from={ac.from}",
 			location: "none",
 			conditions: adminCondition
-		}, {
+		},
+		{
+			key: "spa-deferred-page",
```

Comment:
I don't think we need this one, as the defer page will NOT be rendered in iframe at all from Jira.

Reply:
Removed